lib/mongo/cluster/topology/replica_set.rb

```diff
@@ -213,6 +213,12 @@ def remove_hosts?(description)
         end
 
         # Whether a specific server in the cluster can be removed, given a description.
+				# As described in the SDAM spec, a server should be removed if the server's
```

Comment:
What happened with the whitespace here?

Reply:
Oops, looks like I need to configure tabs vs. spaces properly on my personal laptop